tests/zgoubi-integration-test.sh.template

```diff
@@ -35,7 +33,7 @@ fi
 
 footer=10
 res_file_length=$(wc -l zgoubi.res.expected | awk '{print $1;}')
```

Comment:
just fyi: (( header=$(wc -l < zgoubi.res.expected) - footer ))

Thanks: I didn't know that you didn't need "$" inside (( )).